lib/danger/danger_core/plugins/dangerfile_gitlab_plugin.rb

```diff
@@ -174,11 +165,12 @@ def html_link(paths, full_path: true)
       same_repo = mr_json[:project_id] == mr_json[:source_project_id]
       sender_repo = ci_source.repo_slug.split("/").first + "/" + mr_json[:author][:username]
       repo = same_repo ? ci_source.repo_slug : sender_repo
+      host = @gitlab.host
 
       paths = paths.map do |path|
         url_path = path.start_with?("/") ? path : "/#{path}"
         text = full_path ? path : File.basename(path)
-        create_link("#{repo}/blob/#{commit}#{url_path}", text)
+        create_link("https://#{host}/#{repo}/blob/#{commit}#{url_path}", text)
```

Comment:
We recommend in the docs that you just put in the hostname without that, I agree though 👍